debug_toolbar/panels/history/panel.py

```diff
@@ -0,0 +1,106 @@
+import json
+import logging
+import re
+import sys
+from collections import OrderedDict
+
+from django.conf import settings
+from django.conf.urls import url
+from django.template.loader import render_to_string
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from debug_toolbar.panels import Panel
+from debug_toolbar.panels.history import views
+from debug_toolbar.panels.history.forms import HistoryStoreForm
+
+logger = logging.getLogger(__name__)
+
+SENSITIVE_CREDENTIALS = re.compile("api|token|key|secret|password|signature", re.I)
+CLEANSED_SUBSTITUTE = "********************"
+
+
+def _clean_data(data):
+    """
+    Clean a dictionary of potentially sensitive info before
+    sending to less secure functions.
+    Not comprehensive
+
+    Taken from django.contrib.auth.__init__.credentials
```

Comment:
I was hoping there would be a way to use https://docs.djangoproject.com/en/3.1/ref/settings/#default-exception-reporter-filter for this. I didn't even know there was a setting for this, I assumed it was internal-only. I'd have referenced the setting earlier if I was aware that it existed at all.

(Also, maybe Django should use the `SafeExceptionReporterFilter` instead of reimplementing parts of it in django.contrib.auth.)

Reply:
Ah, good call. I'll rework this to make use of that.

Reply:
I'm thinking I need to remove this. The django implementation only uses it for debug cases which defeats the purpose of this.